src/search/search.stories.ts

```diff
@@ -11,12 +11,19 @@ storiesOf("Search", module).addDecorator(
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
 		template: `
-			<ibm-search [theme]="theme" [placeholder]="placeholder" [disabled]="disabled" [size]="size"></ibm-search>
+			<ibm-search
+				[theme]="theme"
+				[placeholder]="placeholder"
+				[autocomplete]="autocomplete"
+				[disabled]="disabled"
+				[size]="size">
+			</ibm-search>
 		`,
 		props: {
 			size: select("size", ["lg", "sm"], "lg"),
 			theme: select("theme", ["dark", "light"], "dark"),
 			disabled: boolean("disabled", false),
+			autocomplete: text("autocomplete", "on"),
```

Comment:
Maybe a `select` would make more sense here ...
```suggestion
			autocomplete: text("autocomplete", ["off", "on"], "on")
```